og_main_test.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import requests
+
+from bs4 import BeautifulSoup
+
+
+class test_open_graph_standards:
+
+    def test(self, meta, required_tags, found_tags):
```

Comment:
We are almost there. We need a name for this test. a test called test is not a good name. The test should tell what you are testing like "test_sign_into_fxa_new_account" or something. If you were to put 5 tests in a test class and they were named test1-test5 and test3 failed, you would not know what failed just by looking at the name. It would cause you to have to do more digging than necessary.